custom_components/bhyve/binary_sensor.py

```diff
@@ -89,3 +90,41 @@ def _on_ws_data(self, data):
 
     def _should_handle_event(self, event_name, data):
         return event_name in [EVENT_FS_ALARM]
+
+    
+class BHyveTemperatureAlert(BHyveDeviceEntity):
+
+    def __init__(self, hass, bhyve, device):
+        name = "{} temperature alert".format(device.get("name"))
+        super().__init__(hass, bhyve, device, name, "alert")
+
+    def _setup(self, device):
+        self._available = device.get("is_connected", False)
+        self._state = self._parse_status(device.get("status", {}))
```

Comment:
Should add the attributes for the threshold temperatures and any other relevant attributes

Reply:
That’s a great idea, only I dont think they provide this in the API. I dont see it.

Reply:
SCRATCH THAT!! I found it, attributes added as requested.

Reply:
Thanks! Can you please upload your diagnostics for the device so I can look at the properties?